src/lib/output/themes/default/partials/member.declaration.tsx

```diff
@@ -29,12 +29,44 @@ export function memberDeclaration(context: DefaultThemeRenderContext, props: Dec
 
     const visitor = { reflection: renderTypeDeclaration };
 
+    /** Fix for #2717. If type is the same as value the type is omited */
+    function shouldRenderType() {
+        if (props.type && props.type.type === "literal") {
+            const typeObject = props.type.toObject();
+            const value = typeObject.value;
+            if (!value) {
+                // should be unreachable
+                return true;
+            }
+            if (typeof value === "object") {
+                return true;
+            }
+            const reflectionTypeString: string = value.toString();
+            let defaultValue = props.defaultValue!;
```

Comment:
This assertion is unsafe, `defaultValue` may very well be undefined here, which would result in a crash due to using `defaultValue.toString()` below. Given that if it is `undefined` you want that comparison to be false anyways, I'd recommend just getting rid of the `toString` and the non-null assertion here.